doc/reference/std/text/json.md

```diff
@@ -121,6 +121,46 @@ If bound to `list->vector` then JSON lists will be parsed as vectors.
 #("a" 1)
 ```
 
+## json-sort-keys
+``` scheme
+json-object-keys
+```
+
+This is a parameter that can be used to control how JSON objects should be encoded.
+If true (the default), keys in JSON objects represented by hash-tables will be sorted
+before they are printed.
+
+::: tip Examples
+``` scheme
+> (parameterize ((json-object-walist? #f))
+    (hash->list (string->json-object "{\"a\",1}")))
+(("a" . 1))
+
+> (parameterize ((json-object-walist? #t))
+    (string->json-object "{\"a\",1}"))
+(("a" . 1))
+```
+
+## json-object-walist?
+``` scheme
+json-object-walist?
+```
+
+Parameter to control how JSON objects should be decoded.
+If false (the default), JSON objects will be hash-tables.
+If true, JSON objects will be walist (or walistq depending on json-symbolic-keys).
+
```

Comment:
add a sentence about rationale, "this allows you to preserve order" or something.